controller/space.go

```diff
@@ -53,6 +54,26 @@ func (c *SpaceController) Create(ctx *app.CreateSpaceContext) error {
 		if err != nil {
 			return jsonapi.JSONErrorResponse(ctx, err)
 		}
+		/*
```

Comment:
Yeah, I've left out commented stuff in many places which are not relevant - those I will surely remove. Thanks for your vivid reviews :)